tests/test_public_models_predicts.py

```diff
@@ -104,81 +104,77 @@
-        )
+#     for title, model_id, app_id, user_id in TEXT_MODEL_TITLE_IDS_TUPLE:
+#         if title == "translate romance":
+#             request = service_pb2.PostModelOutputsRequest(
+#                 user_app_id=resources_pb2.UserAppIDSet(user_id=user_id, app_id=app_id),
+#                 model_id=model_id,
+#                 inputs=[
+#                     resources_pb2.Input(
+#                         data=resources_pb2.Data(text=resources_pb2.Text(raw=SPANISH_TEXT))
+#                     )
+#                 ],
+#             )
+#         else:
+#             request = service_pb2.PostModelOutputsRequest(
+#                 user_app_id=resources_pb2.UserAppIDSet(user_id=user_id, app_id=app_id),
+#                 model_id=model_id,
+#                 inputs=[
+#                     resources_pb2.Input(
+#                         data=resources_pb2.Data(text=resources_pb2.Text(raw=ENGLISH_TEXT))
+#                     )
+#                 ],
+#             )
+#         response = post_model_outputs_and_maybe_allow_retries(
+#             stub, request, metadata=metadata(pat=True)
+#         )
+#         raise_on_failure(
+#             response,
+#             custom_message=f"Text predict failed for the {title} model (ID: {model_id}).",
+#         )
 
 
-@both_channels
-def test_image_predict_on_public_models(channel):
-    stub = service_pb2_grpc.V2Stub(channel)
+# @both_channels
+# def test_image_predict_on_public_models(channel):
+#     stub = service_pb2_grpc.V2Stub(channel)
 
-    for title, model_id in MODEL_TITLE_AND_ID_PAIRS:
-        request = service_pb2.PostModelOutputsRequest(
-            model_id=model_id,
-            inputs=[
-                resources_pb2.Input(
-                    data=resources_pb2.Data(image=resources_pb2.Image(url=DOG_IMAGE_URL))
-                )
-            ],
-        )
-        response = post_model_outputs_and_maybe_allow_retries(stub, request, metadata=metadata())
-        raise_on_failure(
-            response,
-            custom_message=f"Image predict failed for the {title} model (ID: {model_id}).",
-        )
+#     for title, model_id in MODEL_TITLE_AND_ID_PAIRS:
+#         request = service_pb2.PostModelOutputsRequest(
+#             model_id=model_id,
+#             inputs=[
+#                 resources_pb2.Input(
+#                     data=resources_pb2.Data(image=resources_pb2.Image(url=DOG_IMAGE_URL))
+#                 )
+#             ],
+#         )
+#         response = post_model_outputs_and_maybe_allow_retries(stub, request, metadata=metadata())
+#         raise_on_failure(
+#             response,
+#             custom_message=f"Image predict failed for the {title} model (ID: {model_id}).",
+#         )
 
 
 @both_channels
 def test_video_predict_on_public_models(channel):
     stub = service_pb2_grpc.V2Stub(channel)
 
-    models_with_no_video_support = {
-        "color",
-    }
+    title = "general"
```

Comment:
Looks good, though one thing we do lose is testing all image models in these tests can batch ok. Though I think that's fine because that'll be a model specific error, not platform specific error, someone should catch before even adding a test for it.